civis/ml/tests/test_model.py

```diff
@@ -465,26 +465,60 @@ def test_estimator(mock_le):
         "The Estimator is only downloaded once and cached."
 
 
-@mock.patch.object(_model.cio, "file_id_from_run_output",
-                   mock.Mock(return_value=11, spec_set=True))
+@mock.patch.object(_model.cio, "file_id_from_run_output")
```

Comment:
Could you add `autospec=True` here, and in the other patch decorators you've changed?